wproofreader/values.yaml

```diff
@@ -52,22 +56,37 @@ image:
   # overrides the image tag whose default is the chart appVersion
   tag: ""
 
+# Readiness Options allows customizing the default readiness probe
+readinessProbeOptions: {}
+#  initialDelaySeconds: 10
+#  periodSeconds: 10
+#  timeoutSeconds: 5
+#  successThreshold: 1
+#  failureThreshold: 3
+
+# Liveness Options allows customizing the default liveness probe
+livenessProbeOptions: {}
+#  periodSeconds: 10
+#  timeoutSeconds: 3
+#  failureThreshold: 1
+
 # number of Pods (AppServer instances) to retain; is ignored if HorizontalPodAutoscaling is enabled (see below)
 replicaCount: 1
 
 # HorizontalPodAutoscaling parameters. For more info, see documentation: https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale
 # if enabled, requires "resources" (see below)
-autoscaling:
-  enabled: false
-  minReplicas: 1
-  maxReplicas: 5
-  targetCPUUtilizationPercentage: 80
-  targetMemoryUtilizationPercentage: 80
-
-# max resources usage allowed for pod
+# Note: Wproofreader is able to scale only based on CPU usage metric. The targetMemoryUtilizationPercentage is not supported.
+autoscaling: {}
+#  enabled: false
+#  minReplicas: 1
+#  maxReplicas: 5
+#  targetCPUUtilizationPercentage: 80
+
+# Max resources usage allowed for pod
 # have to be specified if HorizontalPodAutoscaling is enabled
 # if none are given, uses all available on host
 # (to add your values remove "{}" form the line below, uncomment and edit the ones further down)
+# Ref: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/
 resources: {}
 #  limits:
 #    cpu: 100m
```

Comment:
not sure what this if for (allocation) just to make sure our clients don't use these numbers
and for the one below

Reply:
Updated with actual values